recipes/distoptica/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "distoptica" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/distoptica-{{ version }}.tar.gz
+  sha256: bbfbff33237f3e9fc07d3517c9784bb795179d4916d62ecfbcb01709ca707273
+
+build:
+  skip: True  # [win or py<38]
```

Comment:
```suggestion
  noarch: python
```

Reply:
I did that originally but the windows build wouldn't work. The problem has to do with the dependency `pytorch`. For Windows machines, conda-forge only has `pytorch-cpu` available so it fails to build upon searching for `pytorch`. You can see [here](https://anaconda.org/conda-forge/pytorch/) that there is no Windows build for pytorch. Should I proceed nonetheless?

Reply:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.